examples/text-generation/run_generation.py

```diff
@@ -205,6 +206,17 @@ def setup_parser(parser):
         help="Whether to skip hash with views for HPU graphs. When skip_hash_with_views is not used, the input to HPU graphs includes both view and base tensors.",
     )
     parser.add_argument("--verbose_workers", action="store_true", help="Enable output from non-master workers")
+    parser.add_argument(
+        "--simulate_dyn_prompt",
+        default="",
+        type=str,
+        help="If empty static prompt is used. If a comma separated list of integers are passed, we warmup and use those shapes for prompt length",
```

Comment:
I think it should be defined similarly to the `--prompt` arg with default None and type int: https://github.com/huggingface/optimum-habana/blob/4ffb9d7d495347f4aef0d66eecca06bcef4c2b11/examples/text-generation/run_generation.py#L125-L131
It will also return a list and looks cleaner IMO.